networking/README.md

```diff
@@ -0,0 +1,64 @@
+# Robotouille Networking README
+
+## Overview
+
+Built into Robotouille is networking support for multiplayer. Robotouille uses an authoritative server, which clients can connect to. Servers record games played on them to collect data. 
+
+## Available Modes
+
+Several modes are offered, which can be chosen using the `--role` argument:
+
+1. `local`
+2. `server`
+3. `client`
+4. `single`
+5. `replay`
+6. `render`
+
+## Local
+
+This mode runs Robotouille without any networking overhead.
+
+E.g.
+```python main.py --environment_name original```
```

Comment:
try adding `sh` so it looks like a shell command like below (with grave accents ` replacing apostrophe ')
'''sh
python main.py --environment_name original
'''